src/math/quaternion.hpp

```diff
@@ -35,4 +35,11 @@ D angleBetweenQuaternions(const Eigen::Quaternion< D> & q1,
   return theta;
 }
 
+template <typename D>
```

Comment:
Why ? As stated by @jmirabel in
https://github.com/stack-of-tasks/pinocchio/issues/104, it may be usefull to provide this kind of equality check